source/qml/Library/RendererQml/Images/arrowdownIcon.svg

```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><title>navigation/arrow-down_16</title><path d="M8.5001 11.26336438l6.134-6.60428553c.188-.20253009.505-.21355895.707-.02506561.201.18849335.213.50632522.025.7088553l-6.5 6.997314214c-.095.1022676673-.227.159417246-.366.159417246-.139 0-.271-.0571495787-.366-.159417246L1.6311 5.34286854c-.09-.09725454-.134-.21957469-.134-.34189484 0-.13435165.053-.26870329.159-.36696046.202-.18849334.518-.17746448.706.02506561l6.138 6.60428553z" fill="#000" fill-rule="evenodd"/></svg>
```

Comment:
Please pull in the latest code and I have included `arrow-down_12.svg`. Use that instead

Reply:
okay

Reply:
Delete this as we don't need this anymore

Reply:
Sure

Reply:
done